xrmocap/utils/fourdag_utils.py

```diff
@@ -0,0 +1,731 @@
+import math
+import numpy as np
+
+skel_info = dict(
```

Comment:
Some array elements are too long to write as text. Try to save it in a npz file and load the file when needed.
See [gmm_08.pkl](https://github.com/openxrlab/xrmocap/blob/main/xrmocap/model/loss/prior_loss.py#L351) for instance.